### PR TITLE
fix(cdk/testing): Make harnesses click on the center of the element by default

### PR DESCRIPTION
**Pull request by mmalerba**
This is Protractor's default behavior. Prior to this PR we had been
clicking the top-left corner, but that didn't trigger the click event in
Protractor.

fixes https://github.com/angular/components/issues/19002

### CODE REVIEW

File: src/cdk/testing/protractor/protractor-element.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export class ProtractorElement implements TestElement {
     return this.element.clear();
   }
 
-  async click(relativeX = 0, relativeY = 0): Promise<void> {
+  async click(relativeX = 1, relativeY = 1): Promise<void> {
     await browser.actions()
```
Comment (devversion) [inline src/cdk/testing/protractor/protractor-element.ts:77]: Should we click at the center of the elementcby default? This what Protractor did before we added the mouse move. https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/actions_exports_LegacyActionSequence.html#click

**Comment by jelbourn** [comment]
Could you also expand the commit message with more details about the fix?

File: src/material/button/testing/button-harness.e2e.spec.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import {ProtractorHarnessEnvironment} from '@angular/cdk/testing/protractor';
+import {MatButtonHarness} from '@angular/material/button/testing/button-harness';
```
Comment (crisbeto) [inline src/material/button/testing/button-harness.e2e.spec.ts:2]: Shouldn't this be imported from `button/testing`?

File: src/cdk/testing/protractor/protractor-element.ts [hunk 3]
```diff
@@ -73,9 +73,10 @@ export class ProtractorElement implements TestElement {
     return this.element.clear();
   }
 
-  async click(relativeX = 0, relativeY = 0): Promise<void> {
+  async click(...args: number[]): Promise<void> {
+    const offsetArgs = args.length ? [{x: args[0], y: args[1]}] : [];
```
Comment (devversion) [inline src/cdk/testing/protractor/protractor-element.ts:77]: Optional nit: Might be good to have a super-short comment saying that "omitting the last argument of `mouseMove` results in the center of the element being used.

**Comment by mmalerba** [comment]
Yeah this is technically breaking, since I'm not allowing you to pass only `relativeX` anymore, you must pass both or none

**Pull request merged**